config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```
@@ -109,3 +109,20 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-image-builder
       testgrid-tab-name: pr-pull-image-builder-gcp-all
+  - name: pull-goss-populate
+    path_alias: sigs.k8s.io/image-builder
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 20m
+    max_concurrency: 5
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20201021-a62ce8a-master
```

nit: it looks like both this and the gcp job above are using an older image
```suggestion
        - image: gcr.io/k8s-testimages/kubekins-e2e:v20210403-e49d2c6-master
```

---

Fixed. Guess you can tell which job I cut and pasted from. :)